### ISSUE TITLE
Error with particular tbi file

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/tabix-js#11]
```
TypeError: Cannot read property 'compareTo' of undefined
    at TabixIndex._callee4$ (webpack:///./node_modules/@gmod/tabix/dist/tbi.js?:245:35)
    at tryCatch (webpack:///./node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js?:62:40)
    at Generator.invoke [as _invoke] (webpack:///./node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js?:296:22)
    at Generator.prototype.(anonymous function) [as next] (webpack:///./node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js?:114:21)
    at step (webpack:///./node_modules/babel-runtime/helpers/asyncToGenerator.js?:17:30)
    at eval (webpack:///./node_modules/babel-runtime/helpers/asyncToGenerator.js?:28:13) TypeError: Cannot read property 'compareTo' of undefined
    at TabixIndex._callee4$ (tbi.js:245)
    at tryCatch (runtime.js:62)
    at Generator.invoke [as _invoke] (runtime.js:296)
    at Generator.prototype.(/jbrowse/anonymous function) [as next] (webpack:///./node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js?:114:21)
    at step (asyncToGenerator.js:17)
 
```

I get this error with 1.16.1 but in 1.12.5 it worked

Note that for fun this is sort of an interesting use of vcf to store gene expression, not sure what pipeline makes that

![screenshot-localhost-2019 01 04-17-56-34](https://user-images.githubusercontent.com/6511937/50718180-28d23580-104a-11e9-85a1-cb3061287042.png)

https://drive.google.com/open?id=1QI_G7IzTFI8Z-E5kvfX8afJDZWwppoR9

**Comment by rbuels** [comment]
@cmdcolin is this still an issue after the merge of #18?

**Comment by cmdcolin** [comment]
This is now fixed by b58c8a3de2ece762b3d2 it was related to #18 but on the querying side of things e.g. querying outside the range of coordinates tabix allows

**Comment by cmdcolin** [comment]
Follow up issue for this dataset on jbrowse side of things is here though https://github.com/GMOD/jbrowse/issues/1328
